client/src/components/Layout/LandingPage.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import {BsCircleFill} from 'react-icons/bs';
+import img1 from "../../assets/body-img1.png";
+import img2 from "../../assets/body-img2.png";
+import img3 from "../../assets/body-img3.png";
+
+const LandingPage=()=> {
+    return(
+        <div className='bg-black text-white'>
+            <div className='w-[44rem] mx-auto text-center'>
+            <h3 className='text-[2.5rem] font-bold'>Stand out with a free Business Profile</h3>
+            <p className='text-[1.25rem] pt-3 font-thin'>Turn people who find you on this website into new customers with a free Business Profile for your 
+                storefront or service area.Personalise your profile with promotion product photos, offers, posts, and more.</p>
+            </div>
+            <div  className='w-[46rem] mx-auto mt-[4rem] text-center'>
+                <h3 className='text-[2.5rem] font-bold'>First impression is the last impresion</h3>
+                <p  className='text-[1.25rem] pt-3 font-thin'>Highlight essential info and show what makes your business unique, right on your Business Profile.</p>
+            </div>
+            <div className='flex flex-row space-x-[6rem] justify-center pt-[10rem] px-[12rem]'>
```

Comment:
change `pt-[10rem] px-[12rem]` to `py-[11rem] px-[5rem]` and `space-x-[6rem]` to `space-x-[3rem]`